ResearchKit/Common/ORKObserver.m

```diff
@@ -0,0 +1,90 @@
+//
+//  ORKObserver.m
+//  ResearchKit
+//
+//  Created by Ricardo Sánchez-Sáez on 22/04/2015.
+//  Copyright (c) 2015 researchkit.org. All rights reserved.
+//
```

Comment:
Again, proper copyright header. Should be Apple copyright, since this is just a refactor.